File: python/svix/models/app_usage_stats_in.py
# this file is @generated
import typing as t
from datetime import datetime

from .common import BaseModel


class AppUsageStatsIn(BaseModel):
    app_ids: t.Optional[t.List[str]] = None
    """Specific app IDs or UIDs to aggregate stats for.

    Note that if none of the given IDs or UIDs are resolved, a 422 response will be given."""

    since: datetime

    until: datetime

File: kotlin/lib/src/main/kotlin/models/AppUsageStatsIn.kt
// This file is @generated
package com.svix.kotlin.models

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class AppUsageStatsIn(
    val since: Instant,
    val until: Instant,
    /**
     * Specific app IDs or UIDs to aggregate stats for.
     *
     * Note that if none of the given IDs or UIDs are resolved, a 422 response will be given.
     */
    val appIds: Set<String>? = null,
)
